addons/towing/functions/fnc_attachVehicles.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_component.hpp"
+/*
+ * Author: Dystopian
+ * Attaches child to parent vehicle.
+ * Run globally.
```

Comment:
Would be much better if the fnc did the 'Run globally' part automatically.

Reply:
Didn't get you. Do you mean I should replace one global event and one function with 3 events and its own functions (1 for parent owner, 1 global `setTowParent` run and 1 for server)?

Reply:
I mean the script is changed to `Must be run on the server` and the Server runs it globally, otherwise `remoteExec`ed.

Reply:
Still can't understand how it could be better to move script executing to server and split 1 global event to 3 events (1 of those is global anyway) and functions. In **any** case we have to run globally `setTowParent` part. I don't see any profit from function splitting.